def/de/settings.cson

```diff
@@ -36,17 +36,17 @@ Settings:
   ]
   "btn-open-config-folder": "Öffne Konfigurations-Ordner"
   sectionHeadings: [
-    {_label: "Core Settings", value: "Core-Einstellungen"}
+    {_label: "Core Settings", value: "Kern-Einstellungen"}
     {_label: "Editor Settings", value: "Editor-Einstellungen"}
     {_label: "System Settings", value: "Systemeinstellungen"}
     {_label: "Keybindings", value: "Tastenkürzel"}
     {_label: "Installed Packages", value: "Installierte Plugins"}
-    {_label: "Choose a Theme", value: "Ein Theme auswählen"}
-    {_label: "Installed Themes", value: "Installierte Themes"}
+    {_label: "Choose a Theme", value: "Ein Thema auswählen"}
+    {_label: "Installed Themes", value: "Installierte Themen"}
     {_label: "Install Packages", value: "Plugin installieren"}
     {_label: "Featured Packages", value: "Vorgestellte Plugins"}
-    {_label: "Install Themes", value: "Themes installieren"}
-    {_label: "Featured Themes", value: "Vorgestellte Themes"}
+    {_label: "Install Themes", value: "Themen installieren"}
+    {_label: "Featured Themes", value: "Vorgestellte Themen}
```

Comment:
Missing double quote

Reply:
@Henry2o1o  brief steps to fix this PR:

```bash
$ git checkout patch-5

// some editing...

$ git commit -m 'add missing double quote'
$ git push origin
```

then this page will show your new commit